scripts/update-crd-reference/crd.template

```diff
@@ -1,19 +1,44 @@
 ---
-title: {{ .Title }} CRD schema reference
+title: {{ .Title }} CRD schema reference (group {{ .Group }})
 linkTitle: {{ .Title }}
-technical_name: {{ .NamePlural }}.{{ .Group }}
 description: |
 {{- if .Description }}
 {{ .Description | indent 2 }}
 {{- else }}
   Custom resource definition (CRD) schema reference page for the {{ .Title }} resource ({{ .NamePlural }}.{{ .Group }}), as part of the Giant Swarm Management API documentation.
 {{- end }}
 weight: {{ .Weight }}
-source_repository: {{ .SourceRepository }}
-source_repository_ref: {{ .SourceRepositoryRef }}
+crd:
+  name_camelcase: {{ .Title }}
+  name_plural: {{ .NamePlural }}
+  name_singular: {{ .NameSingular }}
+  group: {{ .Group }}
+  technical_name: {{ .NamePlural }}.{{ .Group }}
+  scope: {{ .Scope }}
+  source_repository: {{ .SourceRepository }}
+  source_repository_ref: {{ .SourceRepositoryRef }}
+  versions:
+{{- range .Versions }}
+    - {{ . -}}
+{{- end }}
+  topics:
+{{- range .Topics }}
+    - {{ . -}}
+{{- end }}
+  providers:
+{{- range .Providers }}
+    - {{ . -}}
+{{- end }}
```

Comment:
This places all CRD related data into a common key `.crd`.

For compatibility reasons, existing front matter keys on the top level, like `technical_name`, `source_repository` and `source_repository_ref` are left intact. Once docs_indexer is adapted to the new schema, we can remove the duplication.